target/linux/ramips/base-files/etc/board.d/01_leds

```diff
@@ -205,6 +205,9 @@ hc5661a)
 	ucidef_set_led_netdev "internet" "internet" "$boardname:blue:internet" "eth0.2"
 	set_wifi_led "$boardname:blue:wlan2g"
 	;;
+hc5611)
+	ucidef_set_led_default "system" "system" "$boardname:green:system" "1"
```

Comment:
Drop this LED here. the led is/will be handled by diag.sh and switched to ON after the boot is done.